src/style.css

```diff
@@ -1,3 +1,64 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-size: 12px;
+
+  --hover-color: #00acdc;
+}
+
+header {
```

Comment:
Consider using classes for applying styles. Tag selectors are more suitable for resetting default browser styles or normalizing them

Reply:
Or should I just coomit changes?